src/main/java/com/google/codeu/data/UserMarker.java

```diff
@@ -0,0 +1,27 @@
+package com.google.codeu.data;
+
+public class UserMarker {
+
+  private double lat;
+  private double lng;
+  private String content;
```

Comment:
These three variables should be `private final` since they aren't changed anywhere here.